src/vstart.sh

```diff
@@ -372,6 +373,9 @@ case $1 in
     -e)
         ec=1
         ;;
+    --exporter)
+        exporter=1
```

Comment:
@idryomov Thank you, Ilya! Initializing exporter=0 for consistency makes sense. However, this will require changes across other related PRs where exporter logic is implemented. I'll evaluate the impact and ensure alignment across files before proceeding. Appreciate the feedback!

Reply:
I'm not sure I follow -- which other PRs?  This should be a one-line change.

Reply:
@idryomov My bad Ilya i misinterpreted it. I thought you asked me to change the exporter flag value to 0. I have initialized exporter=0 in the beginning with the rest of the variables. Please check